superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -454,6 +455,41 @@ export default function TableChart<D extends DataRecord = DataRecord>(
         className += ' dt-is-filter';
       }
 
+      type CellClicked = {
+        columnKey: string;
+        rowIndex: number;
+        cellData: DataRecordValue;
+        isAnchor: boolean;
+      };
+
+      const cellClicked = (
+        column: DataColumnMeta,
+        row: Row<D>,
+        value: DataRecordValue,
+      ) => {
+        let isAnchor = false;
+        if (typeof value === 'string') {
+          const parsed = new DOMParser().parseFromString(value, 'text/html');
+          const element = parsed.body.firstChild as HTMLElement;
+          if (element?.tagName === 'A' && element.getAttribute('href') === '#') {
+            const data = element.getAttribute('data');
+            if (data) {
+              try {
+                value = JSON.parse(data);
+                isAnchor = true;
+              } catch (error) {}
+            }
+          }
+        }
+        const msg: CellClicked = {
+          columnKey: column.key,
+          rowIndex: row.index,
+          cellData: value,
+          isAnchor,
+        };
+        SingletonSwitchboard.emit('CellClicked', msg);
```

Comment:
Maybe not now, but I guess it will be good to have a dictionary for emitting names,